src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff.Tool/Program.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+using Microsoft.DotNet.ApiSymbolExtensions.Filtering;
+using Microsoft.DotNet.ApiSymbolExtensions.Logging;
+using Microsoft.DotNet.GenAPI;
+
+namespace Microsoft.DotNet.ApiDiff.Tool;
+
+/// <summary>
+/// Entrypoint for the genapidiff tool, which generates a markdown diff of two
+/// different versions of the same assembly, using the specified command line options.
+/// </summary>
+public static class Program
+{
+    private static readonly string[] defaultAttributesToExclude = new[]
+    {
+        "T:System.AttributeUsageAttribute",
+        "T:System.ComponentModel.EditorBrowsableAttribute",
+        "T:System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute",
+        "T:System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute"
+    };
```

Comment:
IMO it feels wrong to have these defaults defined in the UI layer which should be as thin as possible. Could those be moved into the business layer and used if no attribute option is provided?

Reply:
Done.